src/components/stateless/specific/pages/CreateOfferPage/CreateOfferPage.jsx

```diff
@@ -146,7 +146,7 @@ class CreateOfferPage extends Component {
               <SelectorField className={styles.selector} options={this.props.qualities.map(name => ({ value: name }))}
                              inputRef={el => this.formFields.requirementsName = el}
                              label='Category'/>
-              <span className={styles.paragraph}>or <Link to='create-requirements'>create custom requirements</Link>
+              <span className={styles.paragraph}>or <Link to='create-requirements'>create custom requirements </Link>
```

Comment:
That would create a space in a link, should be space after the link (hint, make it in one line with what you have in a following line.